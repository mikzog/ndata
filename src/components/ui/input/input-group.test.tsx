import React from 'react';
import renderer from 'react-test-renderer';
import InputGroup from './input-group';

describe('InputGroup', () => {
  it('renders without crashing', () => {
    const component = renderer.create(<InputGroup />);
    const json = component.toJSON();

    expect(json).toMatchSnapshot();
  });
});
