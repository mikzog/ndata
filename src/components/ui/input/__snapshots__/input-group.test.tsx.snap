// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputGroup renders without crashing 1`] = `
<div
  className="nd-input-group"
/>
`;
